### PR TITLE
Test werk

### PR DESCRIPTION
**Pull request by matthewrsj**
@MaraJade @Kennric your feedback?

This tests what we have here. If this looks good, we can start creating more tests for future functionality.

### CODE REVIEW

**Comment by MaraJade** [comment]
Looks good to me

File: werkzeug_test.py [hunk 1]
```diff
@@ -1,3 +1,41 @@
...
+import unittest
+from request_handler import Forms, create_msg
+from werkzeug.test import Client
+from werkzeug.testapp import test_app
+from werkzeug.wrappers import BaseResponse, Request
+from werkzeug.test import EnvironBuilder
+from StringIO import StringIO
+
+
+class Test_formsender(unittest.TestCase):
```
Comment (Kennric) [inline werkzeug_test.py]: Classes should be CamelCase, so this should probably be TestFormsender

**Comment by Kennric** [comment]
I think the tests look ok, but it would be very useful to have some comments explaining what is being tested and how. In fact, this is a good way to start writing tests, outline in english what you want to test and what the results should be, then write the actual test code.

**Comment by matthewrsj** [comment]
@Kennric commented, fixed name, and fixed a test that was testing the wrong thing.

**Comment by Kennric** [comment]
+1

**Pull request merged**